### PR TITLE
Add step function to mimotf object

### PR DESCRIPTION
**Pull request by WillemASmit**
This function enables the user to simulate a mimotf objects's time domain step response.

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -501,6 +501,48 @@ def inverse(self):
         inv = (1./detA)*C_T
         return inv
 
+    def step(self, u_input, t_start = 0, t_end = 100, points = 1000):
```
Comment (alchemyst) [inline utils.py]: Check this spacing - keyword arguments should not be spaced.

File: utils.py [hunk 2]
```diff
@@ -501,6 +501,48 @@ def inverse(self):
...
+        
+        tspan:      The time values corresponding with the response data points.
+        G_response: A list of arrays, with a length equal to the amount of 
+                    outputs of the transfer function. The arrays contain the 
+                    step response data of each output.
+        """
+        
+        tspan = numpy.linspace(t_start, t_end, points)
+        
+        G_stepdata = [[0 for i in range(self.shape[0])], [0 for i in \
```
Comment (alchemyst) [inline utils.py]: You don't need this continuation (`\`) - Python allows line breaks in open brackets.

File: utils.py (hunk 1 above)
Comment (WillemASmit) [reply utils.py]: Spacing is fixed

File: utils.py (hunk 2 above)
Comment (WillemASmit) [reply utils.py]: Unnecessary continuation is removed

**Comment by WillemASmit** [comment]
I am not sure why the Travis test is failing. sys.exit(doctest.testmod()[0]) returns 0 when I run the file in a Python console locally.

**Pull request merged**